test/invoice_repo_test.rb

```diff
@@ -14,11 +14,15 @@ def setup
     @se = SalesEngine.from_csv({
                               :invoices => "./data/invoices.csv"
                               })
-    @invoices = @se.invoices
+    # @invoices = @se.invoices
```

Comment:
Let's make sure to omit this comment out if we are in agreeance that the assignment of `@se.invoices` to `@invoices` is not necessary.